src/node_modules.cc

```diff
@@ -313,8 +313,9 @@ const BindingData::PackageConfig* BindingData::TraverseParent(
   return nullptr;
 }
 
-void BindingData::GetNearestParentPackageJSON(
-    const v8::FunctionCallbackInfo<v8::Value>& args) {
+void BindingData::FindNearestParentPackageJSON(
+  const v8::FunctionCallbackInfo<v8::Value>& args
```

Comment:
This change seems to be unrelated

Reply:
Technically yes, this un-mangling is not required to make the change work.